### PR TITLE
Sighting Form - Create Form

### PR DESCRIPTION
**Pull request by jayjay1878**
ws-029: New Sighting Form - Create Form and ws-014: New sighting form endpoint

### CODE REVIEW

File: WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <label for="SpeciesId">Species:</label>
+      <select class="form-control" asp-for="SpeciesId" asp-items="Model.SpeciesList">
+        <option value="">Select a species</option>
+      </select>
+    </div>
+
+    <button type="submit" class="btn btn-primary">Submit</button>
+  </form>
+
+</section>
```
Comment (TimLeach635) [inline WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml:34]: Files should end with exactly one blank line
```suggestion
</section>
```

File: WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <label for="Description">Description:</label>
+      <textarea class="form-control" asp-for="Description"></textarea>
+    </div>
+
+    <div>
+      <label for="SpeciesId">Species:</label>
+      <select class="form-control" asp-for="SpeciesId" asp-items="Model.SpeciesList">
+        <option value="">Select a species</option>
+      </select>
+    </div>
```
Comment (TimLeach635) [inline WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml:29]: Ah nice, did you manage to get this working?

File: WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+@model WhaleSpotting.Sighting
+<section>
+  <form method="post">
```
Comment (TimLeach635) [inline WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml:3]: I think you need to point this form to the right place on the backend with `asp-controller` and `asp-action`

File: WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+@model WhaleSpotting.Sighting
+<section>
```
Comment (TimLeach635) [inline WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml:2]: The other pages set the title here, see if you can copy it from another view and do the same thing, e.g:
```suggestion
@model WhaleSpotting.Sighting
@{
    ViewData["Title"] = "Add a sighting";
}

<section>
```

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
Nice, couple of comments for you!

File: WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml (hunk 2 above)
Comment (jayjay1878) [reply WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml:32]: This what i did originally when we both looked at it, I kept it in there and made a new ticket

File: WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml [hunk 5]
```diff
@@ -0,0 +1,37 @@
+@model WhaleSpotting.Sighting
+@{
+    ViewData["Title"] = "Sightings";
+}
+<section>
```
Comment (TimLeach635) [inline WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml:5]: Blank line separating these please
```suggestion
}

<section>
```

File: WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml [hunk 6]
```diff
@@ -0,0 +1,37 @@
+@model WhaleSpotting.Sighting
+@{
+    ViewData["Title"] = "Sightings";
```
Comment (TimLeach635) [inline WhaleSpotting/Views/Views/Sighting/NewSightingForm.cshtml:3]: I think a better title would be "Add a sighting"

**Review by TimLeach635 — APPROVED** [review]

**Pull request merged**